Morison_Element/README.md

```diff
@@ -2,8 +2,18 @@
 
 **Author:**  	Adam Keester
 
-**Version:** 	WEC-Sim v4.3
-
 **Geometry**	Offshore wind power monopile and tower.
 
-Example on using Traditional Morison Element (no hydro bodies) to run WEC-Sim for a monopile geometry. Execute WEC-Sim in either the v2020a or newer to run this case.  
+**Original Version:** 	WEC-Sim v4.3
+
+**Dependencies:** MATLAB v2020a 
```

Comment:
I cannot remember why this case would require R2020a. There's no broken library links and nothing special in the input file. We can probably remove this dependency since we don't support that far back anyways.